axon-framework/events/event-processors/README.md

```diff
@@ -331,12 +331,12 @@ public class DeadLetterQueueExampleConfig {
 public class DeadLetterQueueExampleConfig {
     
     @Autowired 
-    public ConfigurerModule configureDeadLetterQueueFor(String processingGroup) {
+    public ConfigurerModule configureDeadLetterQueueFor(EventProcessingConfigurer configurer) {
```

Comment:
I don't think we need the `EventProcessingConfigurer` at all, right?
This method returns a `ConfigurerModule`, which is a `Supplier<Configurer>`. 
So, the `configurer` that's used comes natural from the lambda notation.